app/models/court_date.rb

```diff
@@ -14,8 +14,6 @@ class CourtDate < ApplicationRecord
 
   scope :ordered_ascending, -> { order("date asc") }
 
-  DOCX_TEMPLATE_PATH = Rails.root.join("app", "documents", "templates", "default_past_court_date_template.docx")
```

Comment:
We still use the default somewhere I hope
But not here agreed

Reply:
@compwron based off https://github.com/rubyforgood/casa/issues/3216#issuecomment-1634872324, and dpaola2's comment in that thread, that you all would prefer to have all the court templates lean on ONE base template (court_report_template) rather than have one for dates and one for not-dates. if you want to make dates more flexible, can totally add a follow-up, but that was my understanding of the requirements.